.github/workflows/build.yaml

```
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: '14.x'
-      - run: npm install
+      - uses: bahmutov/npm-install@v1
```

Uses https://github.com/marketplace/actions/npm-or-yarn-install-with-caching

The non-action alternative looks like:
```suggestion
      - uses: actions/cache@v2
        with:
          path: ~/.npm
          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
          restore-keys: |
            ${{ runner.os }}-build-${{ env.cache-name }}-
            ${{ runner.os }}-build-
            ${{ runner.os }}-
      - run: npm ci
```

---

Will go ahead and approve. Should we enable any of the rolling cache expiry options?: https://github.com/bahmutov/npm-install#cache-snowballing--rolling-cache-expiry

Took a quick look, but couldn't quickly determine if it would be useful